includes/wikia/tasks/Tasks/CreateNewWikiTask.class.php

```diff
@@ -146,6 +146,9 @@ private function moveMainPage() {
 					'target' => $targetTitle->getPrefixedText(),
 				];
 				if ( $sourceTitle->getPrefixedText() !== $targetTitle->getPrefixedText() ) {
+					$saveUser = $wgUser;
+					$wgUser = \User::newFromName( 'Wikia' );
```

Comment:
Can we put `'Wikia'` into a constant and use it here and in `protectKeyPages` where the same pattern is used?